repo-agent/pkg/llm/provider.go

```diff
@@ -37,7 +37,9 @@ func NewLLMProvider(name string) (Provider, error) {
 	case "gemini-cli":
 		return &Gemini{Executor: &RealCommandExecutor{}}, nil
 	case "claude":
-		return &Claude{}, nil
+		c := &Claude{}
+		c.AddPostProcessor(StripYAMLMarkers)
```

Comment:
We add this in review-sandbox/main.go:89 runReview()
do we need it here as well ?